data/webbrowser.yaml

```diff
@@ -19,7 +19,7 @@ sources:
 - type: ARTIFACT_GROUP
   attributes:
     names:
-    - 'ChromeHistory'
+    - 'ChromeAndDerivedBrowsersHistory'
```

Comment:
why remove ChromeHistory ?

Reply:
The new artifact "ChromeAndDerivedBrowsersHistory" already includes Chrome, so it's more of replacing the "ChromeHistory" artifact with a more complete "Chrome History + derived browsers".

Reply:
ah so ChromeHistory can likely be moved to legacy then https://github.com/ForensicArtifacts/artifacts/issues/455